extractor/src/main/scala/com/digitalasset/extractor/helpers/FutureUtil.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.extractor.helpers
+
+import scalaz.{NonEmptyList, Show, ValidationNel, \/}
+import scala.concurrent.Future
+
+object FutureUtil {
+  def toFuture[E: Show, A](a: E \/ A): Future[A] = {
+    val E = implicitly[Show[E]]
+    a.fold(
+      e => Future.failed(new RuntimeException(E.shows(e))),
```

Comment:
Use `e.shows` instead, comes with `scalaz.syntax.show._`.

Reply:
why? it is just one extra indirect call :) it would deligate to the `E.shows`?